oshi-core/src/main/java/oshi/software/os/linux/LinuxUserInfo.java

```diff
@@ -0,0 +1,37 @@
+package oshi.software.os.linux;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import oshi.util.ExecutingCommand;
+
+public class LinuxUserInfo {
```

Comment:
I'm on the fence whether we need a whole new class for this.  And if we do, I'm thinking maybe something more like `LinuxDatabaseEntries` would be more appropriate (with an eye toward possibly using this class for other `getent` lookups).

If we're just looking up the data every time, we probably don't need the class, but if this class is going to do some sort of timestamp-based tracking of the cache, it's probably best as its own class.